content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -2,12 +2,12 @@
 title: Create a Windows HostProcess Pod
 content_type: task
 weight: 20
-min-kubernetes-server-version: 1.22
```

Comment:
I think the minimum required version is still v1.22 so long as we leave in the comment about needing to have the feature gate enabled.

If someone has a v1.22 cluster and follows this task as documented, including setting the feature gate, will they succeed or fail? If the former, then v1.22 is right.

Reply:
@marosset @jsturtevant do you know? I guess the setup process is the same, but the full feature functionality isn't available in 1.22.

Reply:
Yes this should be 1.22 as you can deploy hpc in 1.22 with the feature gate on.